qutebrowser/completion/completionwidget.py

```diff
@@ -81,10 +81,28 @@ class CompletionView(QTreeView):
         QTreeView:item::hover {
             border: 0px;
         }
-    """
 
-    # FIXME style scrollbar
-    # https://github.com/The-Compiler/qutebrowser/issues/117
+        QTreeView QScrollBar {
+            width: {{ config.get('completion', 'scrollbar.width') }}px;
+            {{ color['completion.scrollbar.bg'] }}
+        }
+
+        QTreeView QScrollBar::handle {
+            /*
+            This is done this way to evade 'magic'
+            ref: http://git.io/vCMqZ
+            Will need to be changed when we utilize jinja better.
+            */
+            background: {{ config.get('colors', 'completion.scrollbar.fg') }};
+            border: {{ config.get('completion', 'scrollbar.padding') }}px solid
+                    {{ config.get('colors', 'completion.scrollbar.bg') }};
```

Comment:
Adding something like `min-height: 10px;` seems to help with the issue I mentioned in the comment.